src/opm/parser/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -859,7 +861,7 @@ void FieldProps::handle_operation(const DeckKeyword& keyword, Box box) {
             continue;
         }
 
-        //throw std::out_of_range("The keyword: " + target_kw + " is not supported");
+        throw OpmInputError("Operation keyword " + keyword.name() + " does not support the keyword " + target_kw, keyword.location());
```

Comment:
This will not use any information from the location variable; I suggest something like:
```
OpmInputError(iformat::fmt("Problem with keyword {{keyword}}\n"
                           "In {{file}} line {{line}}\n"
                           "The target keyword {} is not supported", target_kw), keyword.location());
```
yes I agree it was very unfortunate that the solution based on variadic templates had to go ;-(

Reply:
I've done a fix to this, but delivering it as part of #1989